pkg/sensors/tracing/generic_test.go

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+// Copyright Orange
```

Comment:
We now have an AUTHORS file (https://github.com/cilium/tetragon/blob/main/AUTHORS) can you just use:
> // Copyright Authors of Tetragon

everywhere?

See https://github.com/cncf/foundation/blob/main/copyright-notices.md#copyright-notices for some context.